src/Features/LanguageServer/Protocol/Extensions/ProtocolConversions.cs

```diff
@@ -60,23 +60,13 @@ internal static class ProtocolConversions
             { WellKnownTags.NuGet, LSP.CompletionItemKind.Text }
         };
 
-        /// <summary>
-        /// Workaround for razor file paths being provided with a preceding slash on windows.
-        /// Long term fix in razor here - https://github.com/dotnet/aspnetcore/issues/19948
-        /// </summary>
         public static Uri GetUriFromFilePath(string filePath)
         {
             if (filePath is null)
```

Comment:
I think we may not need this check either anymore - Uri constructor will throw argumentnullexception if we pass a null string

Reply:
We typically leave these null checks in our codebase at top level APIs in the case that the method itself is expanded in the future. Basically it makes it easier to understand where an error happened. i.e. imagine you expand this to later have multiple paths where you construct a URI based on different criteria, you no longer know which one exploded purely based on the call stack.

That being said if you guys don't do that I can remove.

Reply:
At this point we'd just be tagging this file as #nullable enable, and then it's clear null isn't allowed and validated by the compiler at build time.